lib/topological_inventory/ansible_tower/parser.rb

```diff
@@ -37,7 +37,7 @@ def parse_base_item(entity)
     attr_accessor :tower_url
 
     def tower_client_class
-      TopologicalInventory::AnsibleTower::Operations::Core::AnsibleTowerClient
+      TopologicalInventory::AnsibleTower::Operations::Ordering::AnsibleTowerClient
```

Comment:
hm, yeah, I put it under ordering namespace because it's tied to ordering only (at this moment), but it can change in the future, your suggestion makes sense